### PR TITLE
feat(RHOAIENG-24177): Add unit test for Individual Consistency for TrustyAI Python Service

### PR DESCRIPTION
**Pull request by artemsa223**
The test validates the metric's behavior with:
- Real-world data using the bank churn dataset
- Perfect consistency edge case (consistency = 1.0)
- Imperfect consistency with random predictions

### CODE REVIEW

**Review by christinaexyou — CHANGES_REQUESTED** [review]
* Where are you getting the ground truth values from ? All of your tests should use the outputs of [aif360.sklearn.metrics.consistency_score](https://aif360.readthedocs.io/en/stable/modules/generated/aif360.sklearn.metrics.consistency_score.html) as ground truth.

* Rename this PR / commit message so that it aligns with TrustyAI [contributing guidelines](styai-explainability.github.io/trustyai-site/main/saliency-explanations-with-kserve.html).

* Add this unit test to the rest of the fairness metric tests: https://github.com/christinaexyou/trustyai-service/blob/add-fairness-metrics/tests/metrics/test_fairness.py.

File: tests/metrics/test_fairness.py [hunk 1]
```diff
@@ -114,3 +189,51 @@ def test_average_predictive_value_difference():
...
+    score = IndividualConsistency.calculate(
+        proximity_function=proximity_function,
+        samples=X_sample,
+        prediction_provider=prediction_provider
+    )
+
+    assert score == approx(cs_score, abs=0.2)
+
+
+def test_individual_consistency_perfect():
```
Comment (christinaexyou) [inline tests/metrics/test_fairness.py:214]: Per my comment on your previous review, the ground truth should be calculated using `consistency_score` from `aif360.sklearn.metrics`

File: tests/metrics/test_fairness.py [hunk 2]
```diff
@@ -114,3 +189,51 @@ def test_average_predictive_value_difference():
...
+    consistency = IndividualConsistency.calculate(
+        proximity_function=proximity_function,
+        samples=X_sample,
+        prediction_provider=PerfectConsistencyProvider()
+    )
+
+    assert consistency == approx(1.0, abs=1e-5)
+
+
+def test_individual_consistency_imperfect():
```
Comment (christinaexyou) [inline tests/metrics/test_fairness.py:233]: Same comment here as above

**Comment by christinaexyou** [comment]
@artemsa223 lgtm ! Can you squash and rebase your commits so that there's only one commit with the message: "feat( RHOAIENG-24177): Add unit test for Individual Consistency for TrustyAI Python Service"

**Comment by christinaexyou** [comment]
@artemsa223 it looks like there's an unwanted space in your commit message between "feat(" and "RHOAIENG". Can you please fix this, thanks !
